internal/cmd/api-key/command_delete.go

```diff
@@ -32,24 +31,19 @@ func (c *command) delete(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	errs := &multierror.Error{ErrorFormat: errors.CustomMultierrorList}
-	var deleted []string
-	for _, id := range args {
+	deleted, err := deletion.DeleteResources(args, func(id string) error {
```

Comment:
Would it make more sense to put this in the existing `resource` package? 
```suggestion
	deleted, err := resource.Delete(args, func(id string) error {
```

Reply:
Yeah. I went ahead and just moved everything out of that package and into other packages. `resource` already had a `ValidatePrefixes` function so I went ahead and put `ValidateArgs` there as well. `DefaultPostProcess` is only meant to be used with (what is now) `Delete`, so it also went there.

I put the default prompt string functions into form, next to the actual prompting functions themselves.